tests/zlib.rs

```diff
@@ -0,0 +1,48 @@
+#[cfg(all(
+    feature = "static-curl",
+    not(feature = "zlib"),
+    not(feature = "zlib-ng-compat")
+))]
+#[test]
+fn static_with_zlib_disabled() {
+    assert_eq!(curl::Version::get().feature_libz(), false);
+    assert!(curl::Version::get().libz_version().is_none());
+}
+
+#[cfg(all(
+    feature = "static-curl",
+    feature = "zlib",
+    not(feature = "zlib-ng-compat")
+))]
+#[test]
```

Comment:
Instead of having separate tests for this can this use `cfg!(...)` to avoid conditional compilation and always run the tests?

Reply:
ok let me change it